### PR TITLE
Create ADOPTERS.md

### PR DESCRIPTION
**Pull request by ariel-adam**
Adding the list of adopters for CoCo

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Confidential Containers Adopters
+
+This page contains a list of organizations/companies and projects/products who use confidential containers as adopters in different usage levels (development, beta, production, GA etc...) 
+
+**NOTE:** For adding your organization/company and project/product to this table (alphabetical order) fork the repository and open a PR with the required change. 
+See list of adopter types at the bottom of this page.
+
+## Adopters
+
+| Organization/Company                                              | Project/Product                                               | Usage level              | Adaptor type                     | Details                                                                   |
```
Comment (magowan) [inline ADOPTERS.md:10]:
```suggestion
| Organization/Company                                              | Project/Product                                               | Usage level              | Adopter type                     | Details                                                                   |
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+|-------------------------------------------------------------------|---------------------------------------------------------------|--------------------------|----------------------------------|---------------------------------------------------------------------------|
+| [Edgeless Systems](https://www.edgeless.systems/)                 | [Contrast](https://github.com/edgelesssys/contrast)           | Beta                     | Service Provider / Consultancy   | Contrast runs confidential container deployments on Kubernetes at scale.                                |
+| [IBM-Z](https://www.ibm.com/z)                                    | [IBM LinuxONE](https://www.ibm.com/linuxone)                  | Beta                     | Service Provider                 | Confidential Containers with Red Hat OpenShift Container Platform and IBM® Secure Execution for Linux (see [details](https://www.ibm.com/blog/confidential-containers-with-red-hat-openshift-container-platform-and-ibm-secure-execution-for-linux/))  |
```
Comment (magowan) [inline ADOPTERS.md:13]:
```suggestion
| [IBM](https://www.ibm.com/z)                                    | [IBM LinuxONE](https://www.ibm.com/linuxone)                  | Beta                     | Service Provider                 | Confidential Containers with Red Hat OpenShift Container Platform and IBM® Secure Execution for Linux (see [details](https://www.ibm.com/blog/confidential-containers-with-red-hat-openshift-container-platform-and-ibm-secure-execution-for-linux/))  |
```

**Review by magowan — CHANGES_REQUESTED** [review]
Just couple of minor changes suggested,
beyond that looks like a good starter page to kick things off

Thanks

**Comment by ariel-adam** [comment]
@magowan I've added the relevant changes

File: ADOPTERS.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Confidential Containers Adopters
+
+This page contains a list of organizations/companies and projects/products who use confidential containers as adopters in different usage levels (development, beta, production, GA etc...) 
+
+**NOTE:** For adding your organization/company and project/product to this table (alphabetical order) fork the repository and open a PR with the required change. 
+See list of adopter types at the bottom of this page.
```
Comment (Xynnn007) [inline ADOPTERS.md:6]:
```suggestion
See list of adopter types in [Adopter Types][1].

[1]: #adopter-types
```

Then we can have a hyperlink of markdown.

**Review by Xynnn007 — COMMENTED** [review]
Thanks @ariel-adam . only some nits

File: ADOPTERS.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Adopter types 
+
+See CNCF [definition of an adopter](https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter) <br>
+
+Any single company can fall under several categories at once in which case it should enumerate all that apply and only be listed once
+- **End-User** (CNCF member) - Companies and organizations who are End-User members of the CNCF
+- **Another project** - an open source project that leverages a CNCF project as part of their solution, integrates with for compatibility and interoperability,
+  or is used in the supply chain of another project
+- **end users** (non CNCF member)- companies and organizations that are not CNCF End-User members that use the project and cloud native technologies internally, or build upon
```
[the remark this answers is not in the document]
Comment (ariel-adam) [reply ADOPTERS.md:26]: it actually need to be "end users" and not "End users" :-).
Please see the CNCF guidelines https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter

**Review by zvonkok — APPROVED** [review]
LGTM

**Review by sameo — APPROVED** [review]

File: ADOPTERS.md (hunk 3 above)
Comment (ariel-adam) [reply ADOPTERS.md:6]: Any chance we can differ the change to after the basic page was merged?
I'm not familiar with hyperlink markdown within the page and I won't really have time today to look into it and make sure it works

**Review by Xynnn007 — APPROVED** [review]

**Review by magowan — APPROVED** [review]
LGTM

**Pull request merged**